cmd/git/main.go

```diff
@@ -156,6 +158,17 @@ func runGitClone(ctx context.Context) error {
 		}
 	}
 
+	if flagValues.revision == "" {
```

Comment:
I guess this will fix your unit test failures.

```suggestion
	if flagValues.revision == "" && flagValues.resultFileBranchName != "" {
```

Reply:
yeah it does. Thnx